src/BuiltinExtensions/ComfyUIBackend/WorkflowGeneratorModelSupport.cs

```diff
@@ -709,6 +709,14 @@ public void LoadClip3(string type, string modelA, string modelB, string modelC)
                     }, id, false);
                     LoadingModel = [modelNode, 0];
                 }
+                else if (IsZImage())
+                {
+                    string modelNode = CreateNode("NunchakuZImageDiTLoader", new JObject()
+                    {
+                        ["model_name"] = model.Name.EndsWith("/transformer_blocks.safetensors") ? model.Name.BeforeLast('/').Replace("/", ModelFolderFormat ?? $"{Path.DirectorySeparatorChar}") : model.ToString(ModelFolderFormat),
```

Comment:
I imagine they don't have any of the legacy format for the new zimage, so this extra logic can be skipped